rosidl_generator_cpp/resource/msg__struct.hpp.template

```diff
@@ -107,8 +107,8 @@ for field in spec.fields:
 
   // field types and members
 @[for field in spec.fields]@
-  typedef @(msg_type_to_cpp(field.type))
-    _@(field.name)_type;
+  using _@(field.name)_type =
+    @(msg_type_to_cpp(field.type));
```

Comment:
Hmm, I thought the style convention was to indent twice after wrapping, though that might not be the case. Again, these comments are from running uncrustify over the generated headers.